Feature/faster history getitem (#730)
skorch-dev/skorch: CHANGES.md

### Changed

- We no longer pass the `epoch` parameter to LR schedulers, since that parameter has been deprecated. We now rely on the scheduler to keep track of the epoch itself.

> - Changed implementation of `net.history` access to make it faster; this should not have any noticeable effects, but if you encounter bugs, please create an issue

Reviewer: ```suggestion
- Changed implementation of `net.history` access to make it faster, which should be most noticeable for small models/datasets.
```

Author: Yes, thanks, for sure this entry should mention what the benefit is. I included it but didn't use exactly your proposal.
